core/src/main/java/io/kestra/plugin/core/flow/Template.java

```diff
@@ -58,28 +58,30 @@
     examples = {
         @Example(
             full = true,
-            code = {
-                "id: template",
-                "namespace: company.team",
-                "",
-                "inputs:",
-                "  - id: with_string",
-                "    type: STRING",
-                "",
-                "tasks:",
-                "  - id: 1_return",
-                "    type: io.kestra.plugin.core.debug.Return",
-                "    format: \"{{ task.id }} > {{ taskrun.startDate }}\"",
-                "  - id: 2_template",
-                "    type: io.kestra.plugin.core.flow.Template",
-                "    namespace: company.team",
-                "    templateId: template",
-                "    args:",
-                "      my-forward: \"{{ inputs.with_string }}\"",
-                "  - id: 3_end",
-                "    type: io.kestra.plugin.core.debug.Return",
-                "    format: \"{{ task.id }} > {{ taskrun.startDate }}\"\n"
-            }
+            code = """
+                id: template
+                namespace: company.team
+                
+                inputs:
+                  - id: with_string
+                    type: STRING
+                
+                tasks:
+                  - id: 1_return
+                    type: io.kestra.plugin.core.debug.Return
+                    format: "{{ task.id }} > {{ taskrun.startDate }}"
+                
+                  - id: 2_template
+                    type: io.kestra.plugin.core.flow.Template
+                    namespace: company.team
+                    templateId: template
+                    args:
+                      my-forward: "{{ inputs.with_string }}"
```

Comment:
Do we want it to be `my-forward` or `my_forward`?

Reply:
Its an argument and can depend on the template.